examples/pod-spec-patch.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: pod-spec-patch-
+spec:
+  entrypoint: whalesay
+  arguments:
+    parameters:
+      - name: message
+        value: 100m
+
+  templates:
+  - name: whalesay
+    podspecpatch: '{"containers":[{"name":"main", "resources":{"limits":{"cpu": "{{workflow.parameters.message}}"}}}]}'
```

Comment:
message is a poor choice for the name of the parameter. Can we use cpu-limit?

Reply:
Also, does this example work? `podspecpatch` does not match the struct tag for the type definition.